aredis/commands/sorted_set.py

```diff
@@ -7,6 +7,8 @@
                           string_keys_to_dict,
                           int_or_none)
 
+VALID_ZADD_OPTIONS = {'XX', 'NX'}
```

Comment:
According to  the check logic below, `CH` and `INCR` option pass to `zaddoption` function may raise error

Reply:
Yeah, I didn't include CH or INCR logic. Can definitely add it though.